R/class-network-bipartite.R

```diff
@@ -0,0 +1,98 @@
+newBipartiteNetwork <- function(df = data.frame(),
+                                sourceNodeColumn = character(),
+                                targetNodeColumn = character(),
```

Comment:
is there anything from the plot.data class that we'd want here? We could make even a more abstracted class like stats chart